subjects/devops/right/README.md

```diff
@@ -0,0 +1,28 @@
+## right
+
+### Instructions
+
+Create a file `right.sh` that will get the output of a file and parse it, and then write it to a file with a specific format using a single command.
+
+Get the output of the `ls` command, parse it with the `grep` command to filter for files without a `.txt` extension, and write the output to a file called `filtered_files.txt`, check the example bellow:
+
+### Usage
+
+```console
+$ ls
+sample1.txt sample2 sample3.txt sample4
+$ ./right.sh
+$ cat filtered_files.txt
+sample2
+sample4
+$
+```
+
+### Hints
+
+command1 | command2 > output_file
```

Comment:
May could be made cleared if wrapped `command1 | command2 > output_file`
or even a code bloc
```console
command1 | command2 > output_file
```

Reply:
done